src/app/routes/enrolment/enrolment-list/enrolment-list-filter/enrolment-list-filter.component.ts

```diff
@@ -56,4 +49,21 @@ export class EnrolmentListFilterComponent implements OnInit, OnDestroy {
   statusChangeHandler(value: FilterStatus) {
     this.enrolmentFilterListService.setStatus(value);
   }
+
+  public async setFilters(showRevokeFilters: boolean) {
+    this.statusButtons = showRevokeFilters
+      ? [
+          FilterStatus.All,
+          FilterStatus.NotRevoked,
+          FilterStatus.RevokedOffChainOnly,
+          FilterStatus.Revoked,
```

Comment:
I think It's not needed because we will have a Revoked tab (which shows revoked on and off) or just Revoked off chain only, but John said in the story that there wouldnt be a reason someone would revoke on chain only and not off, or it may not be possible to revoke only on.